packages/fluent/scss/card/_variables.scss

```diff
@@ -2,176 +2,180 @@
 @use "../core/" as *;
 @use "../button/_variables.scss" as *;
 
-/// Horizontal padding of the card.
+/// The horizontal padding of the Card.
 /// @group card
 $kendo-card-padding-x: map.get( $kendo-spacing, 3 ) !default;
-/// Vertical padding of the card.
+/// The vertical padding of the Card.
 /// @group card
 $kendo-card-padding-y: map.get( $kendo-spacing, 3 ) !default;
-/// Width of the border around the card.
+/// The width of the border around the Card.
 /// @group card
 $kendo-card-border-width: 0px !default;
-/// Border radius of the card.
+/// The border radius of the Card.
 /// @group card
 $kendo-card-border-radius: $kendo-border-radius-md !default;
+/// The inner border radius of the Card.
+/// @group card
 $kendo-card-inner-border-radius: calc( #{$kendo-card-border-radius} - #{$kendo-card-border-width} ) !default;
-/// Font family of the card.
+/// The font family of the Card.
 /// @group card
 $kendo-card-font-family: var( --kendo-font-family, inherit ) !default;
-/// Font size of the card.
+/// The font size of the Card.
 /// @group card
 $kendo-card-font-size: var( --kendo-font-size-sm, inherit ) !default;
-/// Line height of the card.
+/// The line height of the Card.
 /// @group card
 $kendo-card-line-height: var( --kendo-line-height, normal ) !default;
 
-/// Spacing between the cards in the card deck.
+/// The spacing between the Cards in the Card deck.
 /// @group card
 $kendo-card-deck-gap: map.get( $kendo-spacing, 4 ) !default;
 
-/// Background color of the card.
+/// The background color of the Card.
 /// @group card
 $kendo-card-bg: var( --kendo-component-bg, initial ) !default;
-/// Text color of the card.
+/// The text color of the Card.
 /// @group card
 $kendo-card-text: var( --kendo-component-text, initial ) !default;
-/// Border color of the card.
+/// The border color of the Card.
 /// @group card
 $kendo-card-border: var( --kendo-component-border, initial ) !default;
-/// Shadow of the card.
+/// The shadow of the Card.
 /// @group card
 $kendo-card-shadow: var( --kendo-box-shadow-depth-1, none ) !default;
 
-/// Background color of the card when focused.
+/// The background color of the focused Card.
 /// @group card
 $kendo-card-focus-bg: $kendo-card-bg !default;
-/// Text color of the card when focused.
+/// The text color of the focused Card.
 /// @group card
 $kendo-card-focus-text: null !default;
-/// Border color of the card when focused.
+/// The border color of the focused Card.
 /// @group card
 $kendo-card-focus-border: var( --kendo-component-border, initial ) !default;
-/// Shadow of the card when focused.
+/// The shadow of the focused Card.
 /// @group card
 $kendo-card-focus-shadow: var( --kendo-box-shadow-depth-3, none ) !default;
 
-/// Text size of the card header.
+/// The font size of the Card header.
 /// @group card
 $kendo-card-header-font-size: var( --kendo-font-size-lg, inherit ) !default;
-/// Font family of the card header text.
+/// the font family of the Card header text.
```

Comment:
Make the first letter uppercase and remove the word text. -> The font family of the Card header.